tools/wpt/testfiles.py

```diff
@@ -51,7 +51,7 @@ def git(cmd, *args):
         full_cmd = [u"git", cmd] + list(item.decode("utf8") if isinstance(item, bytes) else item for item in args)  # type: List[Text]
         try:
             logger.debug(" ".join(full_cmd))
-            return subprocess.check_output(full_cmd, cwd=repo_path).decode("utf8").strip()
+            return six.ensure_str(subprocess.check_output(full_cmd, cwd=repo_path).decode("utf8")).strip()
```

Comment:
It seems like this is now changing the type in Python 2.

We have `subprocess.check_output` that's returning bytes. Then `decode` is converting to text, assuming the output is UTF-8. So far so reasonable. Then `ensure_str` does nothing in Python 3 (because `str` is text there) and re-encodes to bytes in Python 2 (because `str` is bytes there). As far as possible I'd like us to use the same types for Py2 and Py3 (the exception is stuff like the stdlib requiring different types for arguments between the two versions).

Reply:
OK makes sense